spec/firebase_app_distribution_helper_spec.rb

```diff
@@ -61,8 +61,13 @@
     end
 
     it 'returns an array when the string passed in has multiple values seperated by commas' do
-      array = helper.string_to_array("string1, string2, string3")
-      expect(array).to eq(["string1", "string2", "string3"])
+      array = helper.string_to_array("string1,string2,string3")
+      expect(array).to eq(%w[string1 string2 string3])
+    end
+
+    it 'returns an array with trimmed values' do
+      array = helper.string_to_array("string1, str ing2 , string3")
```

Comment:
also add leading and trailing spaces? And multiple spaces? E.g. something like `"   string1,   str ing2    , string3  "`

Reply:
Great suggestion. Updated!